### DISCUSSION TITLE
Does renovate bot support "Bitbucket Datacenter"?

### DISCUSSION THREAD

**Discussion by dioniseo** [renovatebot/renovate#8602] [Request Help] [answered]
Hello, there are only Bitbucket Cloud and Bitbucket Server configurations mentioned in the docs. Does anybody know whether Renovate Bot also compatible with Bitbucket Datacenter?

Thanks.

**Comment by viceice** [comment, marked as answer]
Sure, it's the same platform as Bitbucket Server, so use `platform=bitbucket-server` . 🙃

**Reply by dioniseo** [reply]
Got it, thanks!
